### PR TITLE
Trinsic Okapi binaries v1.2.1

### PR DESCRIPTION
**Pull request by fundthmcalculus**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/36768)

### CODE REVIEW

**Comment by fundthmcalculus** [comment]
This is a set of shared libraries which the [Trinsic SDKs](https://github.com/trinsic-id/sdk) consume. There isn't supposed to be an executable, just a couple dynamic link libraries added to the path.

File: manifests/t/trinsic/okapi/1.2.1/trinsic.okapi.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+PackageIdentifier: trinsic.okapi
+PackageVersion: 1.2.1
+PackageLocale: en-US
+Publisher: Trinsic (Streetcred ID)
+PublisherUrl: https://github.com/trinsic-id/okapi/
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: Trinsic Okapi
```
Comment (ItzLevvie) [inline manifests/t/trinsic/okapi/1.2.1/trinsic.okapi.locale.en-US.yaml:12]:
```suggestion
PackageName: okapi
```

File: manifests/t/trinsic/okapi/1.2.1/trinsic.okapi.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.3 $debug=NVS1.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: trinsic.okapi
+PackageVersion: 1.2.1
+PackageLocale: en-US
+Publisher: Trinsic (Streetcred ID)
```
Comment (ItzLevvie) [inline manifests/t/trinsic/okapi/1.2.1/trinsic.okapi.locale.en-US.yaml:7]:
```suggestion
Publisher: Trinsic
```

File: manifests/t/trinsic/okapi/1.2.1/trinsic.okapi.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+InstallerType: msi
+Scope: machine
+InstallerSwitches:
+  Silent: /qn
+  SilentWithProgress: /qb
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/trinsic-id/okapi/releases/download/v1.2.0/okapi-1.2.1-x86_64.msi
+  InstallerSha256: F50962F481F7D9547C9FC7DB48D308BD9FA46C89D5E8F5748A14FF53E7895D0B
```
Comment (ItzLevvie) [inline manifests/t/trinsic/okapi/1.2.1/trinsic.okapi.installer.yaml:17]:
```suggestion
  InstallerSha256: F50962F481F7D9547C9FC7DB48D308BD9FA46C89D5E8F5748A14FF53E7895D0B
  ProductCode: "{DDA9DC0E-851A-4AD3-81D4-26B86BD626A5}"
```

Comment (ItzLevvie) [reply manifests/t/trinsic/okapi/1.2.1/trinsic.okapi.installer.yaml:17]: This must match Windows Registry.

File: manifests/t/trinsic/okapi/1.2.1/trinsic.okapi.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/t/trinsic/okapi/1.2.1/trinsic.okapi.locale.en-US.yaml:12]: This must match Control Panel.

File: manifests/t/trinsic/okapi/1.2.1/trinsic.okapi.locale.en-US.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/t/trinsic/okapi/1.2.1/trinsic.okapi.locale.en-US.yaml:7]: This must match Control Panel.

File: manifests/t/trinsic/okapi/1.2.1/trinsic.okapi.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: Trinsic Okapi
+PackageUrl: https://github.com/trinsic-id/okapi/
+License: Apache License
+LicenseUrl: https://github.com/trinsic-id/okapi/blob/main/LICENSE
+Copyright: Copyright (c) Trinsic
+# CopyrightUrl: 
+ShortDescription: Trinsic Okapi native binaries
```
Comment (ItzLevvie) [inline manifests/t/trinsic/okapi/1.2.1/trinsic.okapi.locale.en-US.yaml:18]:
```suggestion
ShortDescription: Collection of tools that support workflows for authentic data and identity management.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/t/trinsic/okapi/1.2.1/trinsic.okapi.locale.en-US.yaml (hunk 1 above)
Comment (fundthmcalculus) [reply manifests/t/trinsic/okapi/1.2.1/trinsic.okapi.locale.en-US.yaml:12]: Will change in control panel to `Trinsic Okapi`, in our experience, `okapi` isn't unique.

File: manifests/t/trinsic/okapi/1.2.1/trinsic.okapi.locale.en-US.yaml (hunk 2 above)
Comment (fundthmcalculus) [reply manifests/t/trinsic/okapi/1.2.1/trinsic.okapi.locale.en-US.yaml:7]: Will change control panel to match this, good to know, thanks!

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Missing changes:
1) https://github.com/microsoft/winget-pkgs/pull/36768/files#r761942376
2) https://github.com/microsoft/winget-pkgs/pull/36768/files#r761942315

PackageName and Publisher needs to match Control Panel because WinGet heavily relies on this. Incorrect PackageName / Publisher means that people may not be able to upgrade your application (ProductCode should bypass this), but it's still good practice to be consistent with Control Panel.

**Comment by fundthmcalculus** [comment]
@ItzLevvie, those two changes are going to be manual, I'm going to update the installer to represent what we want it to be.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> @ItzLevvie, those two changes are going to be manual, I'm going to update the installer to represent what we want it to be.

Sounds good :)

Make sure to set the PR in DRAFT until you have compiled a newer installer with the fixes.

**Comment by fundthmcalculus** [comment]
Got confirmation from leadership these changes are fine.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**